src/index.html

```diff
@@ -7,8 +7,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo"><img src="/images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item item_one">
+            <a href="#" class="nav__link  is-active">Apple</a>
+          </li>
+          <li class="nav__item item_two">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item item_three">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item item_four">
+            <a href="#"
+            class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
Need to fix it everywhere
<img width="1032" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/d1ee2e0e-9152-4750-83b4-95999ca3fad2">
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```